src/ext/BalExtension/wixext/Xsd/bal.xsd

```diff
@@ -195,6 +195,18 @@
         </xs:annotation>
     </xs:attribute>
 
+    <xs:attribute name="UseCorrectUserCulture" type="YesNoType">
+        <xs:annotation>
+            <xs:documentation>
+                When set to "yes", causes WixStdBA/Prereq BA to use the correct API for determining the user's culture settings. Otherwise, the previous API is used to maintain compatiblity with previous versions of WiX.
```

Comment:
The way this code is written, it doesn't require any APIs not available in Windows 2000. There's no requirement for Vista+ here.